frontend/src/scenes/plugins/PluginError.tsx

```diff
@@ -0,0 +1,54 @@
+import { Button, Popover, Tag } from 'antd'
+import moment from 'moment'
+import React from 'react'
+import { ClearOutlined } from '@ant-design/icons'
+import { PluginErrorType } from '~/types'
+
+function CodeBlock({ children }: { children: React.ReactNode }): JSX.Element {
```

Comment:
Could use the already existing `CodeSnippet` instead maybe?

Reply:
Great idea! Looks much better now:
![image](https://user-images.githubusercontent.com/53387/97869634-1989a500-1d12-11eb-94c9-92281d4dce30.png)